src/main/java/edu/kit/ipd/crowdcontrol/objectservice/config/Deployment.java

```diff
@@ -0,0 +1,8 @@
+package edu.kit.ipd.crowdcontrol.objectservice.config;
+
```

Comment:
please also for the datastruct - who knows what will be added here ... ports etc. maybe.